src/tsserver/nodeServer.ts

```diff
@@ -326,7 +277,7 @@ export function initializeNodeSystem(): StartInput {
 
     let cancellationToken: ts.server.ServerCancellationToken;
     try {
-        const factory = require("./cancellationToken");
+        const factory = require("./cancellationToken.js");

```

Comment:
Hi, I came across this line running [dependency-cruiser](https://www.npmjs.com/package/dependency-cruiser) on the typescript codebase. After building, it's the only require/import that can't be resolved. I notice that this file, `nodeServer`, does not have a `cancellationToken` sibling file - is it possible that this is always throwing (seems like it could be, since the catch block silently catches the error and uses `ts.server.nullCancellationToken`)

Reply:
This file should exist as a sibling. https://unpkg.com/browse/typescript@5.6.3/lib/cancellationToken.js

Are you running this tool on the built code, or the source?

Reply:
(realistically, it's not clear to me why this is a separate file at all anyway)

Reply:
Got it - thanks very much!

Reply:
https://github.com/microsoft/TypeScript/pull/60250 you've inspired me